docs/configuration.md

```diff
@@ -0,0 +1,51 @@
+# Environment Configuration
+
+When developing a Probot App, you will need to have several different fields in a `.env` file which specify environment variables. Here are some common use cases:
+
+```
+WEBHOOK_PROXY_URL='https://smee.io/yourcustomurl'
+# This is used to allow smee to communicate as your Webhook URL
+
+WEBHOOK_SECRET=development #OPTIONAL
+# This is the webhook secret used when creating a GitHub App
+
+APP_ID=1234
+# This is the App ID assigned to your GitHub App
+
+PRIVATE_KEY=SECRETS #OPTIONAL
+# This is your private key; however, this variable is optional, if it is not present Probot will look in your project's directory for the private key.
+
+PRIVATE_KEY_PATH=/path/to/private/key #OPTIONAL
+# This is the path to your private key.
```

Comment:
Could we offer some advice about why thy might want to use this? E.g. "The path to a `.pem` file if your private key is stored outside of your project directory."